pkg/spanner/client.go

```diff
@@ -143,13 +145,29 @@ func (c *Client) TruncateAllTables(ctx context.Context) error {
 		}
 	}
 
-	_, err = c.spannerClient.ReadWriteTransaction(ctx, func(ctx context.Context, txn *spanner.ReadWriteTransaction) error {
-		return txn.BufferWrite(m)
-	})
-	if err != nil {
+	var (
+		wg     = &sync.WaitGroup{}
```

Comment:
we should use multierrors' `Group` instead sync.WaitGroup https://pkg.go.dev/github.com/hashicorp/go-multierror#Group

Reply:
That makes sense!
I'll try it, soon...

Reply:
done!